tldw_Server_API/app/core/TTS/adapters/openai_adapter.py

```
@@ -126,20 +131,60 @@ async def initialize(self) -> bool:
+                    }
+                    # Reuse the same error-mapping logic as normal requests.
+                    await self._generate_complete(headers, payload)
+                    logger.info(f"{self.provider_name}: API key verified during initialization")
+                except TTSAuthenticationError as auth_exc:
+                    logger.error(f"{self.provider_name}: API key verification failed during initialization: {auth_exc}")
+                    self._status = ProviderStatus.ERROR
+                    raise TTSProviderInitializationError(
+                        f"Failed to initialize {self.provider_name}: authentication failed",
+                        provider=self.provider_name,
+                        details={"error": str(auth_exc), "error_type": type(auth_exc).__name__},
+                    ) from auth_exc
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)